README.md

```diff
@@ -1,7 +1,7 @@
-> [v0.11 -> v0.11.2 migration guide >>](MIGRATION_GUIDE.md#011---0112)
+> [v0.11 -> v0.11.3 migration guide >>](MIGRATION_GUIDE.md#011---0113)
 
 <div align="center">
-  <img src="./media/teloxide-logo.png" width="250"/>
+  <img src="https://github.com/teloxide/teloxide/raw/master/media/teloxide-logo.png" width="250"/>
```

Comment:
what's the reason for this?...

Reply:
No logo here: https://crates.io/crates/teloxide.

Reply:
Ah, right, that makes sense. Although this should be changed in `dev`, I think. Since only there the `media` change happened (well, ignoring the accidental merge.)